src/components/Garden/Stake/SideBar.tsx

```diff
@@ -7,8 +7,18 @@ import { formatTokenAmount } from '@utils/token-utils'
 
 import coin from './assets/coin.svg'
 import wallet from './assets/wallet.svg'
+import { TokenType } from '@/types/app'
 
-function Sidebar({ stakeActions, staking, token, onDepositOrWithdraw }) {
+type SidebarType = {
```

Comment:
Maybe camelCase here too?
```suggestion
type SideBarType = {
```

Reply:
Hey Felipe, in this case makes no sense to change, since the component itself is `Sidebar`. So SidebarTypes or SidebarProps is good in my opinion.